sources/deny.toml

```diff
@@ -58,7 +58,11 @@ wildcards = "deny"
 
 deny = [
     { name = "structopt" },
-    { name = "clap", wrappers = [ "cargo-readme" ] },
+
+    # The bottlerocket-settings-sdk uses clap, but plans to migrate to argh
+    # We will provide an exception until the following is resolved:
+    # https://github.com/bottlerocket-os/bottlerocket-settings-sdk/issues/23
+    { name = "clap", wrappers = [ "cargo-readme", "bottlerocket-settings-sdk" ] },
```

Comment:
Can we prioritize this? If this exception goes in then inevitably it will be used to justify another exception.

I don't have a problem with clap as such but I don't think we need more than one argument parsing crate in the distro.